web/client/components/map/openlayers/Map.jsx

```diff
@@ -316,7 +317,7 @@ class OpenlayersMap extends React.Component {
 
     componentWillUnmount() {
         const attributionContainer = this.props.mapOptions.attribution && this.props.mapOptions.attribution.container
-            && document.querySelector(this.props.mapOptions.attribution.container);
+            && (this.props.document || document).querySelector(this.props.mapOptions.attribution.container);
```

Comment:
i would uniform this as leaflet map does, in a getDocument function

Reply:
done